e2e-tests/Jenkinsfile

```diff
@@ -34,7 +35,8 @@ pipeline {
         always {
           archiveArtifacts 'e2e-tests/results/*'
           junit 'e2e-tests/results/*.xml'
-          submitToActiveData('e2e-tests/results/py27.log')
+          submitToActiveData('e2e-tests/results/py27_raw.txt')
+          submitToTreeherder('socorro-tests', 'e2e', 'End-to-end integration tests', 'e2e-tests/results/*', 'e2e-tests/results/py27_tbpl.txt')
```

Comment:
In the PR description, @stephendonner points out how there's already a "socorro".

> We probably shouldn't merge this until we figure out and fix (here) or ask for "socorro-tests" to be a recognized (and perhaps unique?) name in Treeherder.